flybywiresim/manuals
# New Standard Operating Procedures

## Summary of Changes

New Standard Operating Procedures.

## Additional context


A Standard Operating Procedures is a document in which indicates all procedures within normal operations. They are used in all commercial flights.


Discord username (if different from GitHub): Inax#1819

## Testing instructions

1. Open the file A320neo SOP QA1
2. Read the A320neo SOP QA1
3. Note any grammar issues, spelling errors or any other issues.

The testers does not require to verify the accuracy of the content. It has been verified by pilot.


## How to download the PR for QA

In the commits section, press on the file "Add files via upload".
